client/src/components/Announcements.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import Announcement from './Announcement'
+import './css/Announcements.css'
+
+class Announcements extends Component {
```

Comment:
Hmm I think it might be better to rename Announcements, since it is so closely related to Announcement. Try something like AnnouncementsList or anything you feel is better.

Also add in comments for the class.